src/shared/ui/link-custom/link-custom.tsx

```diff
@@ -28,14 +30,22 @@ const linkCustomVariants = cva('', {
   defaultVariants: { variant: 'textLink' },
 });
 
+const externalLinkAttributes = {
+  target: '_blank',
+  rel: 'noopener noreferrer',
+};
+
 export const LinkCustom = ({
   children,
   href,
   icon = <></>,
   className = '',
-  variant,
+  variant = 'textLink',
```

Comment:
no need, as textLink a default variant according to cva

Reply:
cva resolves in classnames, but component should know variant earlier to deside if it should render an external indicator icon

Reply:
Ok then let remove default variant from cva

Reply:
done